src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -15,30 +18,52 @@ namespace Microsoft.CodeAnalysis.Editor.InlineParameterNameHints
     /// </summary>
     internal class InlineParameterNameHintsTag : IntraTextAdornmentTag
     {
+        public const string TagId = "inline parameter name hints";
+
         /// <summary>
         /// Creates the UIElement on call
         /// Uses PositionAffinity.Successor because we want the tag to be associated with the following character
         /// </summary>
         /// <param name="text">The name of the parameter associated with the argument</param>
-        public InlineParameterNameHintsTag(string text)
-            : base(CreateElement(text), removalCallback: null, PositionAffinity.Successor)
+        public InlineParameterNameHintsTag(string text, double lineHeight, TextFormattingRunProperties format)
+            : base(CreateElement(text, lineHeight, format), removalCallback: null, PositionAffinity.Successor)
         {
         }
 
-        private static UIElement CreateElement(string text)
+        private static UIElement CreateElement(string text, double lineHeight, TextFormattingRunProperties format)
         {
-            // Constructs the hint block which gets assigned parameter name, a normal fontstyle, and sets the padding 
-            // space around the block to 0
+            // Constructs the hint block which gets assigned parameter name and fontstyles according to the options
+            // page. Calculates a font size 1/4 smaller than the font size of the rest of the editor
             var block = new TextBlock
             {
-                Text = text + ": ",
+                Text = text + ":",
+                HorizontalAlignment = HorizontalAlignment.Center,
+                VerticalAlignment = VerticalAlignment.Center,
                 FontStyle = FontStyles.Normal,
+                FontFamily = format.Typeface.FontFamily,
+                FontSize = format.FontRenderingEmSize - (0.25 * format.FontHintingEmSize),
                 Padding = new Thickness(0),
+                Foreground = format.ForegroundBrush
             };
 
-            block.Measure(new Size(double.PositiveInfinity, double.PositiveInfinity));
+            // Encapsulates the textblock within a border. Sets the height of the border to be 3/4 of the original 
+            // height. Gets foreground/background colors from the options menu. The margin is the distance from the 
+            // adornment to the text
+            var border = new Border
+            {
+                Child = block,
+                VerticalAlignment = VerticalAlignment.Center,
+                HorizontalAlignment = HorizontalAlignment.Center,
+                Background = format.BackgroundBrush,
+                BorderBrush = format.BackgroundBrush,
+                BorderThickness = new Thickness(1),
```

Comment:
Since the Background and BorderBrush are the same color. I believe the BorderBrush and BorderThickness could just be replaced with a Padding = new Thickness(1)

Reply:
Fixed